sql/dd/impl/bootstrap/bootstrap_ctx.h

```diff
@@ -100,6 +100,7 @@ class DD_bootstrap_ctx {
 
   uint m_did_I_S_upgrade_from = 0;
   uint m_actual_I_S_version = 0;
+  uint m_actual_dd_engine = 0;
```

Comment:
Here and elsewhere use the actual type:
```suggestion
  enum legacy_db_type m_actual_dd_engine = DB_TYPE_UNKNOWN;
```